### DISCUSSION TITLE
different authority between admin and regular user

### DISCUSSION THREAD

**Discussion by owii03** [prolinkinfo/RealEstateCRM#102] [Q&A] [answered]
hi there, May I know, what's the different (scope of access) between the admin and regular user?

**Comment by prolinkinfo** [comment, marked as answer]
Hi @owii03 

Admin user is Administrator who can make system level settings like Adding new module, Add/delete fields, add/remove users and can set access control.

Regular user is one who have access provided by the admin user. For example, if Admin has allowed any regular user to have an access to only Property module then other modules will not be available for him.

Please feel free to ask if you have any doubts.

**Comment by johanwarner1122** [comment]
If you're looking for reliable [property management services ](https://estateagentsdagenham.co.uk/property-management/)in Dagenham, I highly recommend checking out local experts who know the area well. Property management in Dagenham has really improved in recent years, and many agencies offer comprehensive services, from handling tenants to managing repairs and maintenance. It’s always a good idea to choose a property management service with a strong reputation for customer service and local knowledge. If you're a landlord, this could really take the stress out of managing your property!

**Comment by Ameliawarner20** [comment]
With our [guaranteed rent service](https://realestateagentslondon.co.uk/), landlords enjoy fixed monthly payments, no void periods, and hassle-free property management. Get reliable income—stress-free!

**Comment by Ameliawarner20** [comment]
Explore [London real estate](https://propertymanagementcompany.uk/) with expert guidance! From luxury apartments to investment properties, we offer buying, selling, and letting services to help you find the perfect property in the heart of the city.
